### PR TITLE
Adds some of the presentations

### PR DESCRIPTION
**Pull request by agustingianni**
Adds links to antonio's presentation and mine.

### CODE REVIEW

**Comment by alex** [comment]
The presentations are not publicly visible, FYI

**Comment by xcorail** [comment]
And the google docs are not shared publicly, so I thought you wanted to store here in PDF

File: Meetup/2019-11/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# GitHub Security Lab Meet-Up
+
+- Kevin BlackHouse 
+- [Newbie fuzzing gains: Getting the most of your iterations by @antonio-morales](https://docs.google.com/presentation/d/1Dm0rHKC4QTmYF4g2DMCknjF1Gs2SR--hKbuTXsffzhY/edit?usp=sharing)
+- OSS-Fuzz: 3 years of running an automated and continuous fuzzing service by Abishek Arya
+- Using QL to find MIN_INT gotcha by @kevinbackhouse
+- [Bug Hunting with CodeQL by @agustingianni](https://docs.google.com/presentation/d/1Dm0rHKC4QTmYF4g2DMCknjF1Gs2SR--hKbuTXsffzhY/edit?usp=sharing)
```
[the remark this answers is not in the document]
Comment (agustingianni) [reply Meetup/2019-11/README.md]: Oh good point. I did not think about it. Lets see what nico wants to do.

File: Meetup/2019-11/README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # GitHub Security Lab Meet-Up
 
-- Kevin BlackHouse 
+- Kevin BlackHouse
```
Comment (nicowaisman) [inline Meetup/2019-11/README.md]: It's Kevin Backhouse

Comment (nicowaisman) [reply Meetup/2019-11/README.md]: Actually, we need to just remove that line

**Comment by JLLeitschuh** [comment]
Any of these presentations available in a video format?

**Comment by xcorail** [comment]
Sorry @JLLeitschuh we didn't record this meetup 😢
We'll do it next time!

**Pull request merged**